src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -49,12 +50,12 @@ export const AlterationTypeConstants = {
     FUSION: 'FUSION'
 }
 
-interface ExtendedAlteration extends Mutation, GeneMolecularData {
+export interface ExtendedAlteration extends Mutation, GeneMolecularData {
```

Comment:
We have to figure out where we should put these kind of suplemmentary types.  They don't belong just wherever we first use them.